### ISSUE TITLE
Related Resources - 1.4.10 Reflow - At 400%, most of the content of the panel is cut off

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9868]
As a user reliant on the highest zoom setting to perceive content, all text and components should be visible and accessible at 400% zoom and should reflow into a single column. No content should be obscured by other content or cut off by the edge of the page. Users should not be forced to scroll in two dimensions to view content.

**Linked work item:**
#9388  

**Expected Result:**
At 400%, all content should reflow into a single column without any content being cut off, obscured or needing two dimensional scrolling to view.

**Actual Result:**
When the user zooms in to 400% to view the Related Resources panel, most of the left hand side and the bottom of the panel is cut off, including the entire map. 

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/9105d936-1fde-4a16-9d5d-96a8a65cb87d)

**Comment by apeters** [comment]
fixed by #10273

**Comment by imogenroberts** [comment]
RETEST: The user is able to zoom up to 400% and still fully see and access every part of the Resource Report without having the scroll in two dimensions.

**Evidence:**
![Related Resources - 1 4 10 Fixed](https://github.com/archesproject/arches/assets/77161085/b4c5fd0f-31c6-4c41-8955-6eef6f8ba9ec)
